### PR TITLE
32 describe consumergroup show topic

### LINKED ISSUE #32: esque describe consumergroup doesn't show topic for offset summary
This is only for non-verbose mode.

Expected output:
```
>>> esque describe consumergroup mygroup                      
group_id:          mygroup
group_coordinator: mybroker
offsets:
  mytopic (min, max over 50 partitions):
    consumer_offset:      9080383, 9569775
    topic_low_watermark:  2710, 4067
    topic_high_watermark: 9096612, 9586679
    consumer_lag:         15606, 16904
meta:
  group_id:      mygroup
  protocol:      ''
  state:         Empty
  error_code:    0
  protocol_type: consumer
  members:
    []
```

actual output
```
>>> esque describe consumergroup mygroup                      
group_id:          mygroup
group_coordinator: mybroker
offsets:
  consumer_offset:      9080383, 9569775
  topic_low_watermark:  2710, 4067
  topic_high_watermark: 9096612, 9586679
  consumer_lag:         15606, 16904
meta:
  group_id:      mygroup
  protocol:      ''
  state:         Empty
  error_code:    0
  protocol_type: consumer
  members:
    []
```

### PR DESCRIPTION
**Pull request by b-boehne**
fixes https://github.com/real-digital/esque/issues/32
setting consumer-id argument to required seemed sensible, tell me if there's a use case i didn't see

### CODE REVIEW

**Comment by b-boehne** [comment]
Oh, just noticed the tests. Will update my PR later.

File: esque/consumergroup.py [hunk 1]
```diff
@@ -97,7 +97,9 @@ def get_consumer_offsets(self, group_coordinator, consumer_id, topic_assignment,
                     max(old_max, topic_offsets[partition_id][1] - current_offset),
                 )
 
-            return new_consumer_offsets
+            description_template = "{0} (min, max over {1} partitions)"
+            topic_description = description_template.format(topic.decode('utf-8'), len(consumer_offsets[topic].items()))
```
Comment (hfjn) [inline esque/consumergroup.py]: You could do that in one pass with a f-string

```suggestion
            topic_description = f"{topic.decode('utf-8')} (min, max over {len(consumer_offsets[topic].items()} partitions"```

**Comment by hfjn** [comment]
You still have merge conflict. :(

**Comment by swenzel** [comment]
Could it be that this is actually outdated by now?
#53 will improve the `describe consumergroup` command already.

**Comment by hfjn** [comment]
Fixed in #53

**Pull request closed without merging**